CHANGELOG.md

```diff
@@ -77,7 +77,7 @@ If you're using HTML or custom JavaScript, change:
 
 ##### If you’re using Nunjucks
 
-1. Change your `nunjucks.configure` property so that the only GOV.UK Frontend directory it contains is `node_modules/govuk-frontend/`:
+1. Change your `nunjucks.configure` object so that the only GOV.UK Frontend directory it contains is `node_modules/govuk-frontend/`:
```

Comment:
I believe nunjucks.configure is a 'function' to be specific.

To be accurate this should probably be `Change your `nunjucks.configure` options`

Reply:
Ah great, thanks. I think 'function' works best, if that works for you?

Reply:
You're not really changing the function, you're just changing what you pass to it.

How about something like:

```suggestion
1. Change the list of paths passed to `nunjucks.configure` so that the only GOV.UK Frontend path is `node_modules/govuk-frontend/`
```

?

For reference, this is how Nunjucks documentation talks about the configure function:
https://mozilla.github.io/nunjucks/api.html#configure

Reply:
Sounds good, but could we tweak to 'Change the list of paths in...'?

Reply:
For sure, happy for you to revise further as you see fit! :+1:

Reply:
Ace, thanks - this is amended in https://github.com/alphagov/govuk-frontend/pull/1513/commits/46b32a88c1b7da1af2ba697a405dec8a242431ca.